src/Servers/IIS/IIS/test/Common.FunctionalTests/Inprocess/SynchronousReadAndWriteTests.cs

```diff
@@ -22,12 +22,11 @@ public SynchronousReadAndWriteTests(IISTestSiteFixture fixture): base(fixture)
         }
 
         [ConditionalFact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/7341")]
         public async Task ReadAndWriteSynchronously()
         {
-            for (int i = 0; i < 100; i++)
+            var content = new StringContent(new string('a', 100000));
+            for (int i = 0; i < 500; i++)
```

Comment:
How long does this take? This is starting to look like a stress test.

Reply:
~30 seconds versus 25 before.